src/lib/components/workflow-raw-history-link.svelte

```diff
@@ -0,0 +1,30 @@
+<script lang="ts">
+  import Link from '$lib/holocene/link.svelte';
+  import { translate } from '$lib/i18n/translate';
+  import { isCloud } from '$lib/stores/advanced-visibility';
+  import { parameters } from '$lib/stores/events';
+  import { routeForEventHistory } from '$lib/utilities/route-for';
+
+  const { namespace, workflowId, runId } = $parameters;
+
+  $: href = routeForEventHistory({
+    namespace,
+    workflow: workflowId,
+    run: runId,
+  });
+  $: linkToRedirect = `${href}.json`;
+</script>
+
+{#if !isCloud}
```

Comment:
This needs to be !$isCloud (i.e the value of the store, not the store itself)

Reply:
Ahh, I'll get used to svelte syntax shortly.